### ISSUE TITLE
Facing issue when python3.9 -m pip install bigdl-chronos[pytorch]==2.4.0 

### ISSUE THREAD

**Issue by SjeYinTeoIntel** [intel-analytics/ipex-llm#11396]
![image](https://github.com/intel-analytics/ipex-llm/assets/78522263/c2a3da77-198c-4d5e-84fc-2e5915ee73f1)

wonder how should i install this bigdl-chronos[pytorch] to avoid this error?

**Comment by plusbang** [comment]
Hi, @SjeYinTeoIntel , I failed to reproduce this error. I could run successfully the following steps:
```bash
conda create -n install-test python=3.9 setuptools
conda activate install-test
pip install bigdl-chronos[pytorch]==2.4.0
pip install psutil==5.9.5
```

**Comment by SjeYinTeoIntel** [comment]
Hi,
The issue is happened when "python3.9 -m pip install --upgrade pip". pip version will upgrade to 24.1.
Then will facing above issue.

If i fix my pip version as 24.0 as below command, then no issue.
python3.9 -m pip install --upgrade pip==24.0.

Could you try to use latest pip version to see any issue?

**Comment by plusbang** [comment]
> If i fix my pip version as 24.0 as below command, then no issue. 
> python3.9 -m pip install --upgrade pip==24.0.

Please use pip version `24.0`, we currently do not plan to support the latest pip version.

**Comment by SjeYinTeoIntel** [comment]
noted and thanks =)
